f2f-ipv-stub/src/package.json

```diff
@@ -13,6 +13,7 @@
     "node-jose": "2.2.0"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.26.9",
```

Comment:
Can we remove this caret so the version is pinned we just cleaned this up across the other dependencies